src/styles/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.05);
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo {
+  position: absolute;
+  margin: 0 auto;
+  top: 10px;
+  left: 50px;
+}
+
+.nav {
+  // padding-right: 40px;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: block;
+  line-height: 60px;
+  color: #060b35;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding-right: 20px;
+
+  &li:last-child a {
```

Comment:
it doesn't work

Also, you don't need combined selector (li a) here- you have `.nav-link` class for links